person_capture/gui_app.py

```
@@ -2238,7 +2266,23 @@ def _ratio_list_for_profile(profile: str) -> list[str]:
             return crop, rs, profile
 
         fallback_protect = face_protect or subj or base or (bx1, by1, bx2, by2)
-        fallback_ratio = validated_user_ratios[0] if validated_user_ratios else ("1:1" if face_protect is not None else "2:3")
+        fallback_ratio = None
+        for rs in validated_user_ratios:
+            try:
+                rw, rh = parse_ratio(rs)
+                aspect = float(rw) / max(1e-6, float(rh))
+            except Exception:
+                continue
```

Ruff (0.15.11)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)